### PR TITLE
Add HTTP status phrase in reponse for metricsbeat http module (#5470)

### LINKED ISSUE #5470: Support http status phrase in Metricbeat Http module
Add support for HTTP status phrase in addition to HTTP status code using the same naming as in Packetbeat HTTP module

### PR DESCRIPTION
**Pull request by liketic**
Closes #5470

### CODE REVIEW

**Comment by liketic** [comment]
@exekias Yes, thanks. I pushed 09970231e8046a193a0326c879260fc6fd41d788 . Please review again.

**Comment by christiangalsterer** [comment]
Actually I was just to submit a PR as well... seems that @liketic was a little bit faster ;-)
I would like to make one additional suggestion:

* Rename `status_phrase` to just `phrase` to keep the field names harmonised between different beats, here Packetbeat. There is a discussion about this in #5354 and a pending PR #5469.

File: metricbeat/module/http/_meta/fields.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@
               type: keyword
               description: >
                 The HTTP status code
+            - name: status_phrase
+              type: keyword
+              description: >
```
Comment (christiangalsterer) [inline metricbeat/module/http/_meta/fields.yml]: Maybe add an example, e.g. `OK` for a 200 response?

**Comment by christiangalsterer** [comment]
Just curious if you have seen my previous comment on harmonizing  the names.
@ruflin: What do you think?

**Comment by ruflin** [comment]
@christiangalsterer  +1 on harmonizing field names.

**Comment by liketic** [comment]
@christiangalsterer @ruflin I think it's OK to me.

**Comment by ruflin** [comment]
@liketic Could you update the PR accordingly in case @andrewkroh has no objections?

**Comment by liketic** [comment]
@ruflin @christiangalsterer Apologises for the delay. I pushed 5a381dda02afb9a26fa2b30f32c970a6d54f0e44 . Please review again.

**Comment by christiangalsterer** [comment]
LGTM

**Comment by ruflin** [comment]
jenkins, test it

**Pull request merged**